### PR TITLE
Add hugging face content

### PR DESCRIPTION
**Pull request by ahuang11**

### CODE REVIEW

File: content/hugging_face_template.ipynb-meta [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Title: Hugging Face Template 
+Slug: hugging_face_template
+Date: 06/06/2023
+Category: announcement
+Tags: panel
+Author: Philipp Rudiger, Andrew Huang, Sophia Yang
+Summary: Building an interactive ML dashboard in Panel
+Picture: images/
```
Comment (ahuang11) [inline content/hugging_face_template.ipynb-meta:8]: What image should we have here?

Comment (philippjfr) [reply content/hugging_face_template.ipynb-meta:8]: Just an image of the app would be good.

Comment (ahuang11) [reply content/hugging_face_template.ipynb-meta]: Okay, I added the GIF.

**Comment by philippjfr** [comment]
Will deploy this in the morning.

File: content/hugging_face_template.ipynb-meta [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Title: Hugging Face Template 
+Slug: hugging_face_template
+Date: 06/06/2023
+Category: announcement
+Tags: panel
+Author: Philipp Rudiger, Andrew Huang, Sophia Yang
```
Comment (philippjfr) [inline content/hugging_face_template.ipynb-meta:6]: You and Sophia did most of the work:

```suggestion
Author: Andrew Huang, Sophia Yang, Philipp Rudiger
```

**Pull request merged**